MozillaFoundation/foundation.mozilla.org
# preview panel now reflecting changes to related authors

# Description



Link to sample test page: https://foundation-s-tp1-71-rcc-gaxlhx.herokuapp.com/cms/pages/199/edit/
Related PRs/issues: #11918, [JIRA Ticket](https://mozilla-hub.atlassian.net/browse/TP1-71?atlOrigin=eyJpIjoiNjc5NDY1OTI5YjAyNGQ5ZDlkNjdlOTM1MTczZGE2ZjMiLCJwIjoiaiJ9)

Using #11844 for reference, this pull request introduces a `get_preview_template` method and tests for the RCC Detail Page.

The method retrieves updates to "related authors" stored in CMS memory, and uses `previews/libraries/rcc/detail_page.html` to render these changes in the wagtail preview panel.


┆Issue is synchronized with this [Jira Story](https://mozilla-hub.atlassian.net/browse/TP1-383)



## Steps to Test

1. Visit https://foundation-s-tp1-71-rcc-gaxlhx.herokuapp.com/cms/pages/199/edit/ (Credentials: admin2/admin2)
2. Scroll to the related authors section, they should be in the order of 1,2,3.
3. Make an update (reorder/add/delete) to the list of authors. The change should be reflected in the preview panel.
4. Publish the page
5. Visit the live page, the order of the authors should match the order they are set within the CMS.
6. If they are ordered as expected, testing is complete!
7.
